plugins/kotlin/idea/src/org/jetbrains/kotlin/idea/inspections/ReplaceReadLineWithReadlnInspection.kt

```diff
@@ -0,0 +1,54 @@
+// Copyright 2000-2022 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.idea.inspections
+
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.jetbrains.kotlin.config.ApiVersion
+import org.jetbrains.kotlin.idea.KotlinBundle
+import org.jetbrains.kotlin.idea.base.projectStructure.languageVersionSettings
+import org.jetbrains.kotlin.idea.inspections.collections.isCalling
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.kotlin.psi.psiUtil.getQualifiedExpressionForSelectorOrThis
+
+class ReplaceReadLineWithReadlnInspection : AbstractKotlinInspection() {
+    companion object {
+        private val readLineFqName = FqName("kotlin.io.readLine")
+        private val readlnFqName = FqName("kotlin.io.readln")
```

Comment:
Looks like `readlnFqName` and `readlnOrNullFqName` are used only as strings, so constructing an `FqName` is not necessary.